src/util/get-all-keys.js

```diff
@@ -0,0 +1,10 @@
+
+export default function keys(obj, enumerable = false) {
+  let listOfKeys = [
+    ...Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj),
```

Comment:
This will probably turn it into an `Array.from()` call and we currently don't import the polyfill. That said, the `Object` methods should return an `Array` anyways, right? Do we even need to spread here?